src/Model bias analysis.ipynb

```diff
@@ -1504,477 +2504,3954 @@
+       "25       0.987804         0.998360       0.990714  \n",
+       "15       0.987901         0.997752       0.996306  \n",
+       "14       0.988623         0.997767       0.995162  \n",
+       "42       0.990545         0.998818       0.993577  \n",
+       "24       0.991862         0.999574       0.992234  \n",
+       "39       0.992623         0.998412       0.993084  \n",
+       "13       0.992666         0.997323       0.989507  \n",
+       "18       0.992722         0.999003       0.993769  \n",
+       "20       0.993177         0.977495       0.988595  \n",
+       "7        0.993219         0.997255       0.989666  \n",
+       "40       0.993236         0.998647       0.993236  \n",
+       "44       0.993330         0.997343       0.989208  \n",
+       "49       0.993340         0.996564       0.988742  \n",
+       "38       0.993354         0.997627       0.989346  \n",
+       "10       0.993366         0.990186       0.988925  \n",
+       "48       0.993405         0.996595       0.988973  \n",
+       "46       0.993478         0.997614       0.989396  \n",
+       "22       0.993506         0.997466       0.989546  \n",
+       "21       0.993514         0.997622       0.988973  \n",
+       "26       0.995899         0.999579       0.996740  \n",
+       "29       0.995996         0.999737       0.996628  \n",
+       "28       0.996282         0.999353       0.996067  \n",
+       "30       0.996901         0.999535       0.996849  \n",
+       "34       0.996929         0.999744       0.997032  "
       ]
      },
-     "execution_count": 135,
+     "execution_count": 323,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "model_family_name(wiki_model_names)"
+    "_neg_df[['term', 'orig_fnr_mean', 'debias_fnr_mean', 'rand_fnr_mean', 'orig_tnr_mean', 'debias_tnr_mean', 'rand_tnr_mean']].sort_values('orig_tnr_mean')"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 138,
+   "execution_count": 383,
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "Index([u'rev_id', u'text', u'label', u'wiki_cnn_v3_100', u'wiki_cnn_v3_101',\n",
-       "       u'wiki_cnn_v3_102', u'wiki_cnn_v3_103', u'wiki_cnn_v3_104',\n",
-       "       u'wiki_cnn_v3_105', u'wiki_cnn_v3_106', u'wiki_cnn_v3_107',\n",
-       "       u'wiki_cnn_v3_108', u'wiki_cnn_v3_109', u'wiki_cnn_v3_110',\n",
-       "       u'wiki_cnn_v3_111', u'wiki_cnn_v3_112', u'wiki_cnn_v3_113',\n",
-       "       u'wiki_cnn_v3_114', u'wiki_cnn_v3_115', u'wiki_cnn_v3_116',\n",
-       "       u'wiki_cnn_v3_117', u'wiki_cnn_v3_118', u'wiki_cnn_v3_119',\n",
-       "       u'wiki_cnn_v3_1_median', u'wiki_cnn_v3_1_stddev'],\n",
-       "      dtype='object')"
-      ]
-     },
-     "execution_count": 138,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "wiki_cnn_v3_100 EER: 0.74\n",
```

Comment:
Could you change the print message so it is a bit clearer what is happening here. It seems like the same model (e.g. wiki_cnn_v3_100) is being computed multiple times and getting different EERs. However, I think this is due to different terms being looked at? But do we want to compute the EERs for each model for each term? Or just one EER per model across the entire madlibs dataset?

Reply:
Yes, we want the latter. I'm changing that.